Client/Clients/BatchDownloaderClient.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using Client.Interfaces;
+using Client.Models;
+
+namespace Client.Clients
+{
+    class BatchDownloaderClient : IBatchInterface
+    {
+        private IHttpService _httpService;
+
+        public BatchDownloaderClient(IHttpService httpService)
+        {
+            _httpService = httpService;
+        }
+
+        public IEnumerable<Batch> GetBatchStatus()
+        {
+            return new List<Batch>();
+        }
+
+        public bool GetResult()
+        {
+            return false;
+        }
```

Comment:
// alt logiken for batch kommer under Batchclient derfor er dennne classe ikke nødvendigt.

Reply:
Det ved jeg godt.
Men den er også bare temp indtil at du merger dit ind.